components/x-teaser/src/Standfirst.jsx

```diff
@@ -1,6 +1,18 @@
 import { h } from '@financial-times/x-engine';
+import Link from './Link';
 
-export default ({ standfirst, altStandfirst, headlineTesting }) => {
+
+export default ({ standfirst, altStandfirst, headlineTesting, relativeUrl, url, ...props }) => {
 	const displayStandfirst = headlineTesting && altStandfirst ? altStandfirst : standfirst;
-	return displayStandfirst ? <p className="o-teaser__standfirst">{displayStandfirst}</p> : null;
+	const displayUrl = relativeUrl || url;
+	return displayStandfirst ?
+	<p className="o-teaser__standfirst">
+		<Link {...props} url={displayUrl} attrs={{
```

Comment:
Adding another tabbable element will make screen readers repeat the link. I am not sure if that's something we want.
I guess another question is, do we want to make the standfirst clickable or the entire teaser clickable?

Reply:
As discussed, I've added tabIndex = -1 to the standfirst links so that they aren't tab-able